tool/site/client/src/services/test-packs/placeResources.ts

```diff
@@ -138,11 +148,11 @@ export function createPlaceResources(prescriptionType: PrescriptionType, fhirPre
         mode: "instance",
         address: {
           use: "work",
-          line: ["MUSGROVE PARK HOSPITAL"],
-          city: "TAUNTON",
-          postalCode: "TA1 5DA"
+          line: parentOrganisation.address,
+          city: parentOrganisation.city,
+          postalCode: parentOrganisation.postcode
         }
       } as fhir.Location
-    })
+    }]
```

Comment:
we have an `if` and an `else if` but do we need to catch the case in which neither conditional is true?

Reply:
these are the only 2 supported values atm. can see where this is set in `test-packs\index.ts function: getPrescriptionType`